src/session/session_api.c

```diff
@@ -1806,6 +1806,24 @@ __session_timestamp_transaction(WT_SESSION *wt_session, const char *config)
     API_END_RET(session, ret);
 }
 
+/*
+ * __session_timestamp_transaction_commit --
+ *     WT_SESSION->timestamp_transaction_commit method.
+ */
+static int
+__session_timestamp_transaction_commit(WT_SESSION *wt_session, unsigned long long ts)
```

Comment:
Can the second argument be of type wt_timestamp_t, I am not sure.

Reply:
I don't think so, it's in `wt_internal.h` and not referenced in any public docs.